bears/js/JSStandardBear.py

```diff
@@ -16,13 +17,36 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10.0.3'),
+                    NpmRequirement('babel-eslint', '8'),
+                    NpmRequirement('eslint-plugin-flowtype', '2.41.0'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         parser: str='',
+                         globalVars: typed_list(str)=(),
+                         plugin: str='',
+                         typescript: bool=False,
+                         flow: bool=False):
```

Comment:
Like said in the comment before, we don't want settings like, `plugin` or `parser`. See other bears how settings shall look like.

Reply:
the `parser` option will be removed